integration-tests/stateful-pallet-storage/handleItemized.test.ts

```diff
@@ -0,0 +1,223 @@
+        [providerKeys, providerId] = await createProviderKeysAndId();
+        assert.notEqual(providerId, undefined, "setup should populate providerId");
+        assert.notEqual(providerKeys, undefined, "setup should populate providerKeys");
+
+        // Create a schema for Itemized PayloadLocation
+        const createSchema = ExtrinsicHelper.createSchema(providerKeys, AVRO_CHAT_MESSAGE, "AvroBinary", "Itemized");
+        const [event] = await createSchema.fundAndSend();
+        if (event && createSchema.api.events.schemas.SchemaCreated.is(event)) {
+            schemaId = event.data.schemaId;
+        }
+        assert.notEqual(schemaId, undefined, "setup should populate schemaId");
+        // Create non supported schema
+        const createSchema2 = ExtrinsicHelper.createSchema(providerKeys, AVRO_CHAT_MESSAGE, "AvroBinary", "OnChain");
+        const [event2] = await createSchema2.fundAndSend();
+        assert.notEqual(event2, undefined, "setup should return a SchemaCreated event");
+        if (event2 && createSchema2.api.events.schemas.SchemaCreated.is(event2)) {
+            schemaId_unsupported = event2.data.schemaId;
+            assert.notEqual(schemaId_unsupported, undefined, "setup should populate schemaId_unsupported");
+        }
+
+        // Create a MSA for the delegator and delegate to the provider
+        [ , msa_id] = await createDelegatorAndDelegation(schemaId, providerId, providerKeys);
+        assert.notEqual(msa_id, undefined, "setup should populate msa_id");
+    });
+
+    describe("Itemized Storage Tests 😊/😥", () => {
+
+        it("✅ should be able to call applyItemizedAction and apply actions", async function () {
+            
+            // Add and update actions
+            const payload_1 = {
+                "message": "Hello World",
+            }
+            const add_action = {
+                "Add" : payload_1
+            }
+
+            const payload_2 =  {
+                "message": "Hello World Again",
+            }
+
+            const update_action = {
+                "Add" : payload_2
+            }
+
+            let add_actions = [add_action, update_action];
+            let itemized_add_result_1 = ExtrinsicHelper.applyItemActions(providerKeys, schemaId, msa_id, add_actions);
+            await itemized_add_result_1.fundOperation();
+            const [pageUpdateEvent1, chainEvents ] = await itemized_add_result_1.signAndSend();
+            assert.notEqual(chainEvents["system.ExtrinsicSuccess"], undefined, "should have returned an ExtrinsicSuccess event");
+            assert.notEqual(chainEvents["transactionPayment.TransactionFeePaid"], undefined, "should have returned a TransactionFeePaid event");
+            assert.notEqual(pageUpdateEvent1, undefined, "should have returned a PalletStatefulStorageItemizedActionApplied event");
+        }).timeout(10000);
+
+        it("🛑 should fail to call applyItemizedAction with invalid schemaId", async function () {
```

Comment:
Do we really need these simple failure test cases that should be all covered by unit tests?  My understanding was that we are going to use integration tests for more complicated use-cases.